src/model/immutable/DefaultDraftBlockRenderMap.js

```diff
@@ -47,6 +47,12 @@ const DefaultDraftBlockRenderMap: DefaultCoreDraftBlockRenderMap = Map({
   'header-six': {
     element: 'h6',
   },
+  section: {
```

Comment:
These two new block types will have to be added to the `CoreDraftBlockType` type defined [here](https://github.com/facebook/draft-js/blob/585af35c3a8c31fefb64bc884d4001faa96544d3/src/model/constants/DraftBlockType.js#L17).

Please make the required change and update the PR.

Reply:
Thanks @claudiopro  I have made the requested changes.